src/emulator/controller.ts

```diff
@@ -92,14 +106,13 @@ export async function startAll(options: any): Promise<void> {
   // }
   //
   // The list of emulators to start is filtered two ways:
-  // 1) The service must have a top-level entry in firebase.json
+  // 1) The service must have a top-level entry in firebase.json or an entry in the emulators{} object
   // 2) If the --only flag is passed, then this list is the intersection
-  const targets: string[] = filterTargets(options, VALID_EMULATOR_STRINGS);
+  const targets: string[] = filterEmulatorTargets(options);
```

Comment:
again, unnecessary type. I see a bunch coming, I'm just not going to worry about them any more